hasher-matcher-actioner/terraform/dashboard/main.tf

```diff
@@ -0,0 +1,162 @@
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
+
+# Due to limitations in terraform and the aws_cloudwatch_dashboard resource
```

Comment:
There is a beautiful [templatefile](https://www.terraform.io/docs/language/functions/templatefile.html) function which allows you to store the JSON in a separate file and yet do templating with tfvars / outputs.

Reply:
beautiful is a word, while I love templates, unfortunately it doesn't play super well with `jsonencode` because it returns a string value (double encode nightmare) but I found a way to use it.